themes/vue/layout/layout.ejs

```diff
@@ -21,7 +21,7 @@
     <meta property="og:type" content="article">
     <meta property="og:title" content="<%- page.title ? page.title + ' — ' : '' %>Vue.js">
     <meta property="og:description" content="<%- theme.site_description %>">
-    <meta property="og:image" content="https://<%- theme.root_domain %>/<%- url_for("/images/logo.png") %>">
+    <meta property="og:image" content="https://<%- theme.root_domain %>/images/logo.png">
```

Comment:
It's quite lame that OG protocol [doesn't support SVG](https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images) but I'm taking this opportunity to make this consistent with the `twitter:image` meta tag declaration.